dref/models.py

```diff
@@ -227,7 +228,10 @@ class Status(models.IntegerChoices):
         COMPLETED = 1, _('Completed')
 
     created_at = models.DateTimeField(verbose_name=_('created at'), auto_now_add=True)
-    modified_at = models.DateTimeField(verbose_name=_('modified at'), auto_now=True)
+    modified_at = models.DateTimeField(
+        verbose_name=_('modified at'),
+        default=datetime.now, blank=True
```

Comment:
Use auto_now instead of datetime.now
https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.DateField.auto_now